ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -363,6 +367,27 @@ tests:
       IP_STACK: v4
       NUM_EXTRA_WORKERS: "1"
     workflow: baremetalds-assisted-operator-ztp
+- as: e2e-metal-assisted-operator-ztp-periodic-upgrade
+  cron: '@daily'
+  steps:
+    cluster_profile: packet
+    dependencies:
+      ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:candidate
+      ASSISTED_OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:latest
+      INDEX_IMAGE_INSTALL_OVERRIDE: assisted-operator-index-ocm-2.3
+      INDEX_IMAGE_UPGRADE_OVERRIDE: ci-index
+      OPENSHIFT_INSTALL_RELEASE_IMAGE: release:candidate
+    env:
+      DEVSCRIPTS_CONFIG: |
+        EXTRA_WORKER_VCPU=8
+        EXTRA_WORKER_MEMORY=32768
+        EXTRA_WORKER_DISK=120
+        PROVISIONING_NETWORK_PROFILE=Disabled
+        REDFISH_EMULATOR_IGNORE_BOOT_DEVICE=True
+      DISCONNECTED: "false"
```

Comment:
Remove DISCONNECTED option for now since we are not testing upgrade in DISCONNECTED environment in the upgrade flow.